pkg/api/config.go

```diff
@@ -33,11 +33,13 @@ type Config struct {
 	AzureClusterReaderKubeconfig *v1.Config `json:"azureClusterReaderKubeconfig,omitempty"`
 
 	// misc control plane configurables
-	ServiceAccountKey *rsa.PrivateKey `json:"serviceAccountKey,omitempty"`
-	SessionSecretAuth []byte          `json:"sessionSecretAuth,omitempty"`
-	SessionSecretEnc  []byte          `json:"sessionSecretEnc,omitempty"`
-	HtPasswd          []byte          `json:"htPasswd,omitempty"`
-	AdminPasswd       string          `json:"adminPasswd,omitempty"` //TODO: Remove me before GA!
+	ServiceAccountKey     *rsa.PrivateKey `json:"serviceAccountKey,omitempty"`
+	SessionSecretAuth     []byte          `json:"sessionSecretAuth,omitempty"`
+	SessionSecretEnc      []byte          `json:"sessionSecretEnc,omitempty"`
+	HtPasswd              []byte          `json:"htPasswd,omitempty"`
```

Comment:
You seem to be setting this inside the master script only under testing so it's probably worth to mention that this is also used only during testing.

Reply:
We want to remove the osadmin and stop using the HtPasswd.  Should I put a comment here denoting that its `# only created while RUNNING_UNDER_TEST`?

Reply:
Maybe? Definitely worth adding RUNNING_UNDER_TEST in env.example if it's not already there.

Reply:
Will do.